### PR TITLE
SCANOSS test fixes

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -20,7 +20,8 @@ reporter/src/funTest/assets/*-expected-*	text eol=lf
 reporter/src/main/resources/**/*.ftl	text eol=lf
```
Comment (nnobelis) [inline .gitattributes:21]: i don't understand how this change is relevant to the bug ?!
Why a change to a Reporter test would have an impact on a Scanner test ?

Comment (sschuberth) [reply .gitattributes:21]: I'm not sure what you're referring to, I'm not changing a reporter-test related line in this file. I'm just adding a line for `scanner/src/test/assets/scanoss/filesToScan/*.kt` (and adjusting the indentation in the line above).

Comment (nnobelis) [reply .gitattributes:21]: Yes, it was a mistake from me..

**Review by nnobelis — APPROVED** [review]

**Comment by sschuberth** [comment]
Note that `funTest-analyzer-docker` has actually [passed](https://github.com/oss-review-toolkit/ort/actions/runs/3104576344/jobs/5029209495#step:6:938), just reporting that back does not seem to have worked. So merging nonetheless.

**Pull request merged**